src/pages/character.tsx

```diff
@@ -0,0 +1,87 @@
+import { useNavigate, useParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { APICharacter, unknownCharacter } from '../models/constants';
+import Spinner from '../components/spinner';
+import type { CharacterType } from '../models/types';
+
+const Character: React.FC = () => {
+  const { id } = useParams<{ id: string }>();
+  const [character, setCharacter] = useState<CharacterType | null | undefined>(undefined);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState<string | null>(null);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    setLoading(true);
+    setError(null);
+    fetch(`${APICharacter}/${id}`)
+      .then((res) => {
+        if (!res.ok) {
+          throw new Error('404');
+        }
+        return res.json();
+      })
+      .then((data) => {
+        setCharacter(data);
+      })
+      .catch(() => {
+        setError('Nothing was found for your request');
+        setCharacter(null);
+      })
+      .finally(() => {
+        setLoading(false);
+      });
+  }, [id]);
+
+  if (loading) return <Spinner />;
+  if (character === undefined) return null;
+
+  if (character === null)
+    return (
+      <div>
+        <p>{error ?? ''}</p>
+      </div>
+    );
+
+  return (
+    <div className="mt-4">
+      <div className="rounded-sm border border-solid border-blue-800 bg-white p-4">
```

Comment:
add deleted from home page classes
```suggestion
    <div className="bg-white p-8  mt-4">
      <div className="w-[20vw] rounded-sm border border-solid border-blue-800 bg-white p-4">
```